### PR TITLE
Add ThirdParty json file

### PR DESCRIPTION
**Pull request by baothientran**
This is similar to Cesium for Unreal's ThirdParty json file except for MikkTSpace

### CODE REVIEW

File: ThirdParty.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    "url": "https://github.com/Amanieu/asyncplusplus",
+    "license": "MIT"
+  },
+  {
+    "name": "Catch2",
+    "url": "https://github.com/catchorg/Catch2",
+    "license": "Boost Software License 1.0"
+  },
+  {
+    "name": "cesium-native",
```
Comment (shehzan10) [inline ThirdParty.json:13]: @baothientran This should be removed.

File: ThirdParty.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    "url": "https://github.com/google/draco",
+    "license": "Apache 2.0"
+  },
+  {
+    "name": "earcut",
+    "url": "https://github.com/mapbox/earcut.hpp",
+    "license": "ISC License"
+  },
+  {
+    "name": "Font Awesome Icons",
```
Comment (shehzan10) [inline ThirdParty.json:43]: @baothientran We don't include Font Awesome in Cesium Native

**Review by shehzan10 — CHANGES_REQUESTED** [review]
@baothientran Two changes. Otherwise looks good.

File: ThirdParty.json (hunk 2 above)
Comment (baothientran) [reply ThirdParty.json]: Updated

**Comment by shehzan10** [comment]
Thanks @baothientran!

**Pull request merged**